src/view_wrapper_test.tsx

```diff
@@ -168,7 +172,11 @@ describe('ViewWrapper', () => {
           ),
         });
 
-        ContainerA = withEnvironment(environment({ effects: testEffect, dispatcher }), {
+        ContainerA = withEnvironment(environment({
+          effects: testEffect,
+          dispatcher,
+          renderer: props => React.createElement(ViewWrapper, props),
```

Comment:
why not import the `renderer` from `./react`?

Reply:
Laziness?